checker/build.xml

```diff
@@ -297,6 +288,11 @@
             <package name="org.checkerframework.common.*"/>
             <package name="org.checkerframework.checker.*"/>
 
+            <fileset dir="${checker.loc}/${src}">
+                <include name="**/*.java"/>
+                <exclude name="${checker.jdk8orhigher.sources}" if="${jdk7orlower}"/>
```

Comment:
That's what I'm suggesting https://github.com/typetools/checker-framework/pull/950#issuecomment-253926070
We can't include Opt in a JDK 7 only build, as javadoc would fail.